odoc.opam

```diff
@@ -37,12 +37,9 @@ description: """
 odoc is part of the [OCaml Platform](https://ocaml.org/docs/platform), the recommended set of tools for OCaml.
 """
 
-pin-depends: [
-  ["odoc-parser.dev" "git+https://github.com/ocaml-doc/odoc-parser.git#f98cfe3"]
-]
 
 depends: [
-  "odoc-parser" {>= "2.0.0"}
+  "odoc-parser" {= version}
```

Comment:
Thinking for later: Now that the sources are in the same repository, nothing (except the complicated dune rules required) prevent us from distributing the parser in two different packages `odoc.parser` and `odoc-parser`. `odoc` would no longer depend on `odoc-parser`, which would only be defined by a complicated `(copy ...)` rule.

Reply:
Yep there are definitely some interesting options open in the future.